api/src/iot/import_utils_apis.py

```diff
@@ -0,0 +1,480 @@
+    'verkeersonderzoek_met_cameras': f'{API}KAARTLAAG=PRIVACY_OVERIG&THEMA=privacy',
+    'beweegbare_fysieke_afsluiting': f'{API}KAARTLAAG=VIS_BFA&THEMA=vis',
+}
+
+
+def import_api_data(api_name: str) -> Tuple[List[Exception], int, int]:
+    """
+    takes the name of the api as a param, calls the url that belongs to it in the
+    API_MAPPER. when the data is fetched from the api, it will convert it to a dict
+    and pass it to the convert_api_data together with the api_name.
+    A tuple will be returned that will contain the results. The results will be as follow:
+    (err, inserts, updates)
+    """
+    try:
+        api_url = API_MAPPER[api_name]  # get the url that belongs to the api.
+        response = requests.get(url=api_url)
+
+        # check if response is 200 and content type is json, if not, return
+        if response.status_code != 200 or \
+                'application/json' not in response.headers["Content-Type"]:
+            raise RuntimeError(f"{response.status_code} - {response.content}")
+
+        data = response.json()  # get the content of the response as dict
+        return convert_api_data(api_name=api_name, api_data=data)
+
+    except Exception as e:
+        return ([e], 0, 0)
+
+
+def convert_api_data(api_name: str, api_data: dict) -> tuple:
+    """
+    takes the api_name to find the parser, and the api_data to migrate the data with the
+    existing data. The parser will return a generator of SensorData.
+    from the generator, use the import_person_data and import_senso_data.
+    """
+    parser = PARSERS_MAPPER[api_name]
+
+    sensors = list(parser(api_data))
+    owners_list = [sensor.owner for sensor in sensors]
+    # return f'ERROR importing data from api {api_name}: {sensors}'
+
+    for owner in owners_list:
+        owner.validate()
+
+    imported_owners = {
+        person_data.email.lower(): import_utils.import_person(person_data)
+        for person_data in owners_list
+    }
+
+    counter = Counter()
+
+    errors: List[Exception] = []
+    for sensor_data in sensors:
+        try:
+            import_utils.validate_sensor(sensor_data)
+            owner = imported_owners[sensor_data.owner.email.lower()]
+            _, created = import_utils.import_sensor(sensor_data, owner)
+            counter.update([created])
+        except Exception as e:
+            errors.append(e)
+
+    # delete sensors from the same owner that are not in the api
+    delete_not_found_sensors(sensors=sensors, email=owners_list[0].email)
+
+    return (errors, counter[True], counter[False])
+
+
+def parse_wifi_sensor_crowd_management(data: dict) -> Generator[SensorData, None, None]:
+    """
+    convert the crowd management data list of dictionaries into SensorData objects and
+    yield it. Yield only the sensors with Soort == WiFi sensor.
+    """
+    # create the personData from static defined data because the api doesn't provide
+    # all the required fields
+
+    personData = PersonData(
```

Comment:
nitpick: beter om snake_case te gebruiken hier (dit geldt ook voor de variable in andere functies)